rocAL/source/meta_data/text_file_meta_data_reader.cpp

```diff
@@ -33,7 +33,7 @@ THE SOFTWARE.
 #include "exception.h"
 
 void TextFileMetaDataReader::init(const MetaDataConfig &cfg, pMetaDataBatch meta_data_batch) {
-    _path = cfg.path();
```

Comment:
Why can't it be _path, like how it was before?

Reply:
_path is source path